### DISCUSSION TITLE
Gitsigns opts not working

### DISCUSSION THREAD

**Discussion by alexislayvu** [LazyVim/LazyVim#2411] [Q&A] [answered]
When I uncomment those lines of code, the opts I set no longer work. Does anyone know a fix or a reason why?

<img width="865" alt="Screenshot 2024-01-22 at 5 16 12 PM" src="https://github.com/LazyVim/LazyVim/assets/54639928/e1857b8d-c21a-4ac2-a44a-6f052ecae9bc">

**Comment by jyuan0** [comment, marked as answer]
The `config` function is missing an `opts` parameter, and `opts` should be passed to `gitsigns.setup`. Try this:
```lua
config = function(_, opts)
  local gitsigns = require("gitsigns")
  local scrollbar_gitsigns = require("scrollbar.handlers.gitsigns")
  gitsigns.setup(opts)
  scrollbar_gitsigns.setup()
end,
```

**Comment by alexislayvu** [comment]
That did the trick, thanks! :)
